README.md

```diff
@@ -124,16 +127,73 @@ The final piece is the `Resource`, which is nothing more than a struct that enca
 * HTTP headers
 * HTTP method
 
+#### Configuration
+
+For extra flexibility, you can use the `CoreConfiguration` and `FlowConfiguration` protocols. 
+
+The `CoreConfiguration` protocols defines how the Reactor behaves:
+
+```swift
+public protocol CoreConfiguration {
+/// If the entire flow should fail, when `saveToPersistenceFlow` fails.
+/// `true` by default.
+var shouldFailWhenSaveToPersistenceFails: Bool { get }
+/// If the `saveToPersistenceFlow`, should be part of the flow.
+/// Should be `false` when the flow shouldn't
+/// wait for `saveToPersistenceFlow` to finish (for example it takes
+/// a long time).
+/// Note: if you set it as `false` and it fails, the failure will be
+/// lost, because it's not part of the flow, but injected instead .
+/// `true` by default.
+var shouldWaitForSaveToPersistence: Bool { get }
+}
+```
+
+The `FlowConfiguration` protocol defines how the Reactor Flow is created:
+
+
+```swift
+public protocol FlowConfiguration {
+/// If persistence should be used.
+/// `true` by default.
+var usingPersistence: Bool { get }
+/// If reachability should be used.
+/// `true` by default.
+var shouldCheckReachability: Bool { get }
+/// If the parser should be strick or prune the bad objects.
```

Comment:
```
/// If the parser should be strict or prune the bad objects.
/// Pruning will simply remove objects are not parsable, instead
/// of erroring the flow. Strict on the other hand as soon as it finds
/// a bad object will error the entire flow.
/// Note: if you receive an entire batch of bad objects, it will default to
/// an empty array, which leads to not knowing if the server has no results or
/// if all objects are badly formed.
/// `true` by default.
```
Replace the code comment with the above :smile: